boa/src/exec/throw/mod.rs

```diff
@@ -0,0 +1,8 @@
+use super::{Executable, Interpreter};
+use crate::{builtins::value::ResultValue, syntax::ast::node::Throw};
+
+impl Executable for Throw {
+    fn run(&self, interpreter: &mut Interpreter) -> ResultValue {
```

Comment:
This is just so that debug builds inline this function, it should be small enough so that it doesn't cause any build delays, and should help.
```suggestion
impl Executable for Throw {
    #[inline]
    fn run(&self, interpreter: &mut Interpreter) -> ResultValue {
```